### PR TITLE
Add WebDriver Bidi section for 120 release notes

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

**Comment by nchevobbe** [comment]
cc @whimboo

File: files/en-us/mozilla/firefox/releases/120/index.md [hunk 1]
```diff
@@ -86,6 +86,11 @@ This article provides information about the changes in Firefox 120 that affect d
 
 ### WebDriver conformance (WebDriver BiDi, Marionette)
 
+#### WebDriver BiDi
+
+- Added serialization support for `Proxy` and `Generator` objects ([Firefox bug 1841786](https://bugzil.la/1841786)).
+- Added `authChallenges` property (the list of authentication challenges present in the headers), to `responseStarted` and `responseCompleted` network events, which will be useful in order to handle the upcoming `network.authRequired` event ([Firefox bug 1855149](https://bugzil.la/1855149)).
+
 #### General
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/120/index.md:94]: Please remove these empty sections.

**Review by whimboo — COMMENTED** [review]
Looks good to me. Thanks @nchevobbe. Just one nit.

**Comment by bsmth** [comment]
Just a note, CI is failing because we have two `#### WebDriver BiDi` sections. This will be resolved after empty sections are removed.

<details>

https://github.com/mdn/content/actions/runs/6927498246/job/18841472115?pr=30347

```
  Summary: 1 error(s)
  files/en-us/mozilla/firefox/releases/120/index.md:98 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "#### WebDriver BiDi"]
  error Command failed with exit code 1.
```

</details>

File: files/en-us/mozilla/firefox/releases/120/index.md (hunk 1 above)
Comment (nchevobbe) [reply files/en-us/mozilla/firefox/releases/120/index.md]: done :)

**Comment by nchevobbe** [comment]
> Just a note, CI is failing because we have two `#### WebDriver BiDi` sections. This will be resolved after empty sections are removed.

ah right, this should be fixed now

File: files/en-us/mozilla/firefox/releases/120/index.md [hunk 2]
```diff
@@ -88,10 +88,11 @@ This article provides information about the changes in Firefox 120 that affect d
...
 ### WebDriver conformance (WebDriver BiDi, Marionette)
 
-#### General
-
 #### WebDriver BiDi
 
+- Added serialization support for `Proxy` and `Generator` objects ([Firefox bug 1841786](https://bugzil.la/1841786)).
+- Added `authChallenges` property (the list of authentication challenges present in the headers), to `responseStarted` and `responseCompleted` network events, which will be useful in order to handle the upcoming `network.authRequired` event ([Firefox bug 1855149](https://bugzil.la/1855149)).
+
 #### Marionette
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/120/index.md:97]: I would even remove the empty Marionette section.

```suggestion
```

Comment (nchevobbe) [reply files/en-us/mozilla/firefox/releases/120/index.md]: ah yes, sorry, I removed it now

**Review by whimboo — COMMENTED** [review]
Thanks. That looks fine now. @bsmth can you please review and merge if it looks fine to you as well? Thanks!

**Review by bsmth — APPROVED** [review]
Thanks, both. All looks good to me; merging shortly 👍🏻

**Pull request merged**